app/styles/ember-frost-table/_frost-table.scss

```diff
@@ -3,12 +3,27 @@
 //
 
 .frost-table {
-  table-layout: fixed;
```

Comment:
I believe we still want the _default_ for the table body to be `fixed` - it's more performant than `auto` and flex is entirely different